utils/fasm/src/lut.h

```diff
@@ -0,0 +1,63 @@
+#ifndef LUT_H
+#define LUT_H
+
+#include "netlist_writer.h"
+
+namespace fasm {
+
+// Utility class to create a LUT initialization.
+class Lut {
+ public:
+  // Initialize an LUT of a given number of inputs.
+  Lut(size_t num_inputs);
+
+  // SetOutput sets the lut to output value when the inputs match.
+  //
+  // By default the output from the LUT is always false.
+  void SetOutput(const std::vector<vtr::LogicValue> &inputs, vtr::LogicValue value);
+
+  // Create a wire from input_pin to LUT output.
```

Comment:
This is normally called a `route through` lut. Took me a while to figure out what you meant by "create a wire from a LUT".